app/src/main/res/values/strings.xml

```diff
@@ -80,6 +86,10 @@
     <string name="contact_deleted_description">The contact has been deleted!</string>
     <string name="contact_doesnt_exist">The contact doesn\'t exist</string>
     <string name="contact_already_deleted">It looks like the contact was already deleted.</string>
+    <string name="contact_iban">IBAN</string>
```

Comment:
This is to generic to be listed under `<!-- Contacts -->`, can you list it under `<!-- Words -->` and change its name to "iban"